### PR TITLE
Tiny change: some nicer arrows for the paginated box widget

### PR DESCRIPTION
**Pull request by strouble**
As the title says - i find those arrows nicer:
PaginatedBoxWidget_navigation

### CODE REVIEW

**Comment by strouble** [comment]
> I am not entirely sure, as this arrow is used in all other paginations as well.
> Would break th UX if the arrow changes from one screen to the other.

Ok, got a point. But, i wouldn't see these arrows (in the dashboard widget box) as "standard" pagination arrows, rather as buttons for the prev/next slide. Since they also doesn't reload a whole page - only a small part of it.

And to be honest, I didn't notice the page arrows before, but I found the box arrows weird.

Conclusion: in my opinion, i don't think that the box widget arrows break UX, 'cause they do not load another page.

As an another proposal, one could also change the page arrows.

File: assets/js/widgets/KimaiPaginatedBoxWidget.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ export default class KimaiPaginatedBoxWidget {
 
         this.widget.on('click', '.box-tools a', function (event) {
             event.preventDefault();
-            self.loadPage(jQuery(event.target).attr('href'));
+            self.loadPage(jQuery(event.currentTarget).attr('href'));
```
[the remark this answers is not in the document]
Comment (strouble) [reply assets/js/widgets/KimaiPaginatedBoxWidget.js]: This is necessary with the addtional `span` element in the prev/next buttons. With that, the target is set to the `span` element if i managed klick on it and in that case the `href` falls back to the current page url, which is "/dashboard", which in turn loads the wrong data.

But the `currentTarget` is still the `a` element, even when i klick on the `span` element.

[the remark this answers is not in the document]
Comment (strouble) [reply assets/js/widgets/KimaiPaginatedBoxWidget.js]: From the MDN:
> It always refers to the element to which the event handler has been attached, as opposed to Event.target, which identifies the element on which the event occurred and which may be its descendant.

**Comment by kevinpapst** [comment]
Alright. But lets make it re-usable by registering a `right` and `left` definition: https://github.com/kevinpapst/kimai2/blob/master/src/Twig/IconExtension.php#L20
and then use the icon filter `<i class="{{ 'right'|icon }}"></i>`. I used <i> everywhere, so I think we should stick with it if possible.

**Comment by strouble** [comment]
Ok, added the new icons

**Comment by kevinpapst** [comment]
The test failure is my mistake... we can ignore that one. I will push a fix later on

File: assets/js/widgets/KimaiPaginatedBoxWidget.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (kevinpapst) [reply assets/js/widgets/KimaiPaginatedBoxWidget.js]: Thanks, I didn't now that. Guess I have to check the other JS as well...

**Comment by kevinpapst** [comment]
Thanks @strouble 🚀

**Pull request merged**